client/state/ui/language/reducer.js

```diff
@@ -17,7 +17,7 @@ import { localeSlugSchema } from './schema';
  *
  */
 export const localeSlug = createReducer(
-	'en',
+	false,
```

Comment:
@Tug @yurynix this change caused the initial state for this reducer to not match its schema.  We fixed this in #21020 and we were wondering if we can also change the default from `false` to `null` since this would to a cleaner schema (see discussion on that PR).  Can you take a look?

Reply:
I've took a look at the code, and switching to `null` should be ok IMO.

Reply:
Made the change at: https://github.com/Automattic/wp-calypso/pull/21092